include/psa/crypto.h

```diff
@@ -16,6 +16,14 @@
 #include "crypto_platform.h"
 #endif
 
```

Comment:
In order to merge any external contribution, we need a [signoff line in the Git commit messages](https://github.com/Mbed-TLS/mbedtls/pull/9952/checks?check_run_id=36664114106) to indicate that we are legally allowed to use the contribution. Please edit the commit messages to add a signoff line.